src/graphql/resolvers/backgroundResolver.ts

```diff
@@ -50,23 +57,33 @@ const Background = {
       },
       true
     ),
-  starting_equipment_options: async background =>
-    background.starting_equipment_options.map(async option =>
-      resolveChoice(option, {
-        equipment_category: await EquipmentCategoryModel.findOne({
-          index: option.from.equipment_category.index,
-        }).lean(),
-      })
-    ),
-  ideals: async background =>
-    resolveChoice(background.ideals, {
-      options: background.ideals.from.options.map(async option => ({
-        ...option,
-        alignments: await AlignmentModel.find({
-          index: { $in: option.alignments.map(a => a.index) },
-        }).lean(),
-      })),
+  starting_equipment_options: async (background: Background) =>
+    background.starting_equipment_options.map(async option => {
+      if ('equipment_category' in option.from) {
```

Comment:
Not sure if this `if` is necessary. Also, since it's inside a `map` function, it's a bit strange seeing that if the if failed the function wouldn't return a value.